src/hls-adapter.js

```diff
@@ -245,6 +250,7 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
     this._hls.on(Hlsjs.Events.LEVEL_SWITCHED, this._onLevelSwitched.bind(this));
     this._hls.on(Hlsjs.Events.AUDIO_TRACK_SWITCHED, this._onAudioTrackSwitched.bind(this));
     this._hls.on(Hlsjs.Events.FPS_DROP, (e, data) => this._onFpsDrop(data));
+    this._hls.on(Hlsjs.Events.MEDIA_ATTACHED, () => this._hlsMediaAttachedResolver());
```

Comment:
I think the rest of the event handlers follow a convention so lets stick to it `onMediaAttached`